action_plugins/cli.py

```diff
@@ -98,6 +98,7 @@ def run(self, tmp=None, task_vars=None):
             command = self._task.args['command']
             parser = self._task.args.get('parser')
             engine = self._task.args.get('engine', 'command_parser')
+            key_name = self._task.args.get('key_name')
```

Comment:
Instead of `key_name` can you please keep the term same as `name`. 
Also please add documentation for the new option here https://github.com/ansible-network/network-engine/blob/0d2cff2fd046234c7c0254d50aca456713ba8fec/action_plugins/cli.py#L14-L47

Reply:
I'm looking into this more, should we call it register to match what we typically do when assigning keys in ansible?  Let me know if you have a preference and I'll update it.

Reply:
@jwoogee textfsm_parser module uses the parameter `name` to return `ansible_facts` https://github.com/ansible-network/network-engine/blob/devel/library/textfsm_parser.py#L46.
We don't want to call it `register` as it might create confusion in case there is a necessity of using `register` from ansible in the same task.

Reply:
Thank you, I've changed the term to 'name'.
Would you be able to help with PR https://github.com/ansible-network/cisco_ios/pull/51 or do you know who I can assign as reviewer on that one?

Reply:
Yes I can definitely help.